modules/candidate_parameters/README.md

```diff
@@ -0,0 +1,51 @@
+# Candidate Parameters
+
+## Purpose
+
+The Candidate Parameters module allows users to input 
+candidate data that is not timepoint specific.
+
+## Intended Users
+
+The primary types of users are:
+1. Data entry personnel
+2. Site coordinators validating the data
+
+## Scope
+
+The Candidate Parameters module stores all information 
+for a candidate that are not timepoint dependent. All data 
+being collected during a visit should be entered as an instrument 
+at the visit level.
+
+## Permissions
+
+The Candidate Parameters module uses the following permissions. Any one of them
+is sufficient to have access to the module.
+
+ - candidate_parameter_view
+    - This permission gives the user viewing access to the module and all its tabs.
+
+ - candidate_parameter_edit
+    - This permission gives the user editing access to the module and all its tabs.
+
+## Configurations
+
+The candidate parameters module has the following configurations that affect its usage
+
+ - useProband 
+    - This setting displays/hides the Proband tab of the module
+
+ - useFamilyID
+    - This setting displays/hides the Family tab of the module
+
+ - useConsent 
+    - This setting displays/hides the Consent tab of the module
+
```

Comment:
I would add dots at the end of each phrase to be consistent with Permissions details.

Reply:
done

Reply:
@ridz1208 missing the full stops above